opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -280,3 +300,28 @@ message Status {
   // The status code.
   StatusCode code = 3;
 }
+
+// SpanFlags represents constants used to interpret the
+// Span.flags field, which is protobuf 'fixed32' type and is to
+// be used as bit-fields. Each non-zero value defined in this enum is
+// a bit-mask.  To extract the bit-field, for example, use an
+// expression like:
+//
+//   (span.flags & SPAN_FLAGS_TRACE_FLAGS_MASK)
+//
+// See https://www.w3.org/TR/trace-context-2/#trace-flags for the flag definitions.
+//
+// Note that Span flags were introduced in version 1.1 of the
+// OpenTelemetry protocol.  Older Span producers do not set this
+// field, consequently consumers should not rely on the absence of a
+// particular flag bit to indicate the presence of a particular feature.
```

Comment:
@tigrannajaryan does this mean that flags is not a good option to solve #484? Or is the plan to use two bits to encode whether parent is remote?

Reply:
Use 2 bits, e.g. 00 - Unknown, 01 - IsNotRemote, 10 - IsRemote.